### PR TITLE
Prepare for 2018 Season

### PR DESCRIPTION
**Pull request by axelclark**
Need to make updates for 2018 season
See #283

### CODE REVIEW

File: web/models/fantasy_team.ex [hunk 1]
```diff
@@ -99,10 +99,12 @@ defmodule Ex338.FantasyTeam do
         and (r.status == "active" or r.status == "injured_reserve"),
       right_join: p in assoc(r, :fantasy_player),
       inner_join: s in assoc(p, :sports_league),
+      inner_join: ls in assoc(s, :league_sports),
+       on: ls.fantasy_league_id ==^fantasy_league_id and
```
Comment (credoci) [inline web/models/fantasy_team.ex]: There are spaces around operators most of the time, but not here.

**Comment by axelclark** [comment]
Ebert has finished reviewing this Pull Request and has found:

* 1 possible new issue (including those that may have been commented here).
* 4 fixed issues! 🎉

You can see more details about this review at https://ebertapp.io/github/axelclark/ex338/pulls/284.

**Comment by axelclark** [comment]
Ebert has finished reviewing this Pull Request and has found:

* 4 fixed issues! 🎉

You can see more details about this review at https://ebertapp.io/github/axelclark/ex338/pulls/284.

**Comment by axelclark** [comment]
Ebert has finished reviewing this Pull Request and has found:

* 1 possible new issue (including those that may have been commented here).
* 8 fixed issues! 🎉

You can see more details about this review at https://ebertapp.io/github/axelclark/ex338/pulls/284.

**Comment by axelclark** [comment]
Ebert has finished reviewing this Pull Request and has found:

* 1 possible new issue (including those that may have been commented here).
* 9 fixed issues! 🎉

You can see more details about this review at https://ebertapp.io/github/axelclark/ex338/pulls/284.

**Comment by axelclark** [comment]
Ebert has finished reviewing this Pull Request and has found:

* 9 fixed issues! 🎉

You can see more details about this review at https://ebertapp.io/github/axelclark/ex338/pulls/284.

**Pull request merged**